README.md

```diff
@@ -50,13 +55,13 @@ Note: the following assumes an existing WooCommerce extension with an existing `
 * Install required packages as development dependencies.
 
 ```bash
-npm install --save-dev @woocommerce/e2e-environment @woocommerce/e2e-utils jest
+npm i --save-dev @woocommerce/e2e-environment @woocommerce/e2e-utils @woocommerce/api @wordpress/e2e-test-utils jest
```

Comment:
`@wordpress/e2e-test-utils` no longer needs to be installed. All of it's functionality is exposed in `@woocommerce/e2e-utils`

Reply:
Thanks! Given that, do you think it's still worth keeping this bullet point at the top of the document?

```
* [**WordPress E2E Utils**](https://www.npmjs.com/package/@wordpress/e2e-test-utils) - All of this package is exported from `@woocommerce/e2e-utils`. However, you can install it directly if you choose.
```